From f134276f0f243a2c6896f1eb8e5de79aecb8977b Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Mon, 2 Jul 2012 23:07:25 +0300
Subject: [PATCH 3/3] workaround uuagc's inability to handle 'setup configure
 --enable-tests'

Fixes build tests build failure:

> Preprocessing test suite 'spl-test-hunit' for SPL-0.0.0.920...
>
> CPP/CompileTools.hs:3:18:
>     Could not find module `Bar'
>     Perhaps you meant
>       Var (needs flag -package ghc-7.4.2)
>       Bag (needs flag -package ghc-7.4.2)
>     Use -v to see a list of the files searched for.

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 SPL.cabal |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/SPL.cabal b/SPL.cabal
index 61d7d03..9515789 100644
--- a/SPL.cabal
+++ b/SPL.cabal
@@ -22,9 +22,8 @@ Executable spl-test-hunit-exe
   Build-Depends:  base == 4.*, containers == 0.*, HUnit == 1.*, bytestring == 0.*, hoopl >= 3.8.7.4, mtl == 2.*, parsec >= 3.1, directory == 1.*, QuickCheck == 2.*
   Other-Modules:  Bar
 
-Test-Suite spl-test-hunit
+Executable spl-test-hunit
   Main-Is: Test/Main.hs
-  Type: exitcode-stdio-1.0
   Default-Language: Haskell2010
   Build-Depends:  base == 4.*, containers == 0.*, HUnit == 1.*, bytestring == 0.*, hoopl >= 3.8.7.4, mtl == 2.*, parsec >= 3.1, directory == 1.*, QuickCheck == 2.*
   Other-Modules:   Bar
-- 
1.7.8.6

